src/test/kotlin/uk/gov/justice/digital/hmpps/personrecord/test/messages/CourtCommonPlatformMessages.kt

```diff
@@ -114,11 +118,11 @@ private fun personDefendant(commonPlatformHearingSetup: CommonPlatformHearingSet
   "personDefendant": {
     "personDetails": {
       "address": {
-        "address1": "13 Wind Street",
-        "address2": "Cardiff",
-        "address3": "Wales",
-        "address4": "UK",
-        "address5": "Earth",
+        "address1": "${commonPlatformHearingSetup.address?.buildingName}",
+        "address2": "${commonPlatformHearingSetup.address?.buildingNumber}",
+        "address3": "${commonPlatformHearingSetup.address?.thoroughfareName}",
+        "address4": "${commonPlatformHearingSetup.address?.dependentLocality}",
+        "address5": "${commonPlatformHearingSetup.address?.postTown}",
         "postcode": "CF10 1FU"
```

Comment:
not part of the change I know but would be great to see this hard-coded postcode gone as well